lms/lmsdb/models.py

```diff
@@ -8,6 +8,7 @@
     Type, Union, cast,
 )
 
+from flask_babel import gettext
```

Comment:
We should import it as `_` to make it less noisy (it's a known convention)

Look at example of greater projects, such as [Apache's Incubator Superset](https://github.com/apache/incubator-superset/blob/a6101f72c96d89e3fbf9999f16cf3b5147f73d8f/superset/utils/core.py).